src/renderers/shaders/ShaderLib.js

```diff
@@ -299,7 +299,7 @@ ShaderLib.physical = {
 			sheen: { value: 0 },
 			sheenColor: { value: new Color( 0x000000 ) },
 			sheenColorMap: { value: null },
-			sheenRoughness: { value: 0 },
+			sheenRoughness: { value: 1 },
```

Comment:
The exact values do actually not matter. It's only the type which is important so that something like `uniform.value.copy()` works.

Anyway, for consistency it's probably better to apply the actual default values. You can also update `specularIntensity`.

Reply:
correct. i'm extending the MeshPhysicalMaterial to achieve light rim...